tests/pr_test.go

```diff
@@ -114,3 +114,23 @@ func TestRunUpgradeCompleteExample(t *testing.T) {
 		assert.NotNil(t, output, "Expected some output")
 	}
 }
+
+func TestRunBasicExampleWithFlavor(t *testing.T) {
+	t.Parallel()
+
+	options := testhelper.TestOptionsDefaultWithVars(&testhelper.TestOptions{
+		Testing:            t,
+		TerraformDir:       "examples/basic",
+		Prefix:             "postgres",
```

Comment:
Let's give this a unique prefix so we can spot this test resources, like "postgres-flv"? Or similar?